templates/default/sv-sentry-cron-log-run.erb

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
```

Comment:
I added new sv- templates for the new sentry commands (run|web|cron vs start|celery).
Similarly to the logic above, I wanted to make a clear distinction between versions, rather than patch existing templates. Also similar to how each recipe has a clear if/else block corresponding to < 8.0 & >= 8.0